lib/active_merchant/billing/gateways/cyber_source.rb

```diff
@@ -739,6 +739,9 @@ def add_stored_credential_options(xml, options = {})
         return unless options[:stored_credential]
 
         xml.tag! 'subsequentAuthFirst', 'true' if options.dig(:stored_credential, :initial_transaction)
+
+        return if options.dig(:stored_credential, :network_transaction_id).nil?
+
         xml.tag! 'subsequentAuthTransactionID', options.dig(:stored_credential, :network_transaction_id) if options.dig(:stored_credential, :initiator) == 'merchant'
```

Comment:
Let's stay consistent with the rest of this file:

```suggestion
        network_transaction_id = options.dig(:stored_credential, :network_transaction_id)

        xml.tag! 'subsequentAuthTransactionID', network_transaction_id if network_transaction_id && options.dig(:stored_credential, :initiator) == 'merchant'
```